Mathlib/Order/KrullDimension.lean

```diff
@@ -318,6 +319,21 @@ lemma height_eq_coe_iff {x : α} {n : ℕ} :
     rename_i y _
     cases height y <;> simp; norm_cast; omega
 
+/-- The elements of finite height `n` are the minimial elements among those of height `≥ n`. -/
+lemma height_eq_coe_iff_minimal_le_height (a : α) (n : ℕ) :
```

Comment:
```suggestion
lemma height_eq_coe_iff_minimal_le_height {a : α} {n : ℕ} :
```

Reply:
Ha, seems my grepping for `_iff (` was insufficient.